docs/Developers/Writing_Native_Bears.rst

```diff
@@ -20,6 +20,11 @@ directory of the coala-tutorial repository.
     If you want to wrap an already existing tool, please refer to
     :doc:`this tutorial instead<Writing_Linter_Bears>`.
 
+.. note::
+
+    Also refer `Python3 regex guide <https://docs.python.org/3/library/re.html>`_ for understanding regex.
```

Comment:
for ~~understanding~~ more information on ~~regex~~ regular expressions.

Reply:
You should also concatenate the following sentence with this one by just writing "<linkA> and <linkB>"